app/controllers/glossary_terms/images_controller.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+# Clicking on an image currently fires a GET to these actions... because it
+# comes from a link made by thumbnail_helper#thumbnail(link: url_args)
+# with CRUD refactor, change thumbnail helper to fire a POST somehow?
+
+module GlossaryTerms
+  class ImagesController < ApplicationController
+    before_action :login_required
+
+    before_action :login_required
```

Comment:
Suggestion: Remove duplicate lines
```suggestion
    before_action :login_required
```

Reply:
Hah! Good suggestion :)